distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -309,11 +343,26 @@
     public static final String SAMPLE_SOAP =
             "<?xml version=\"1.0\"?><soap:Envelope xmlns:soap=\"http://schemas.xmlsoap.org/soap/envelope/\"><soap:Body><helloWorld xmlns=\"http://ddf.sdk/soap/hello\" /></soap:Body></soap:Envelope>";
 
+    @Override
+    protected Option[] configureSystemSettings() {
```

Comment:
This configuration will not get picked up here when this test is run in the suite. Your options are to either integrate these configs with the current ones in AbstractIntegrationTest and verify they don't mess up any other tests or to move this test into the ContainerPerClassItestSuite. The latter will impact build time much more that the former. I don't think there should be a problem updating the AbstractIntegrationTest with your configurations.